src/Pages/Main.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import PrivateRoute from '../Routes/PrivateRoute';
+import Home from '../Pages/Home';
+import Login from '../Pages/Login';
+import About from '../Pages/About';
+import categories from '../Pages/categories';
+import Jewellery from '../categories/jewellery';
+import Electronics from '../categories/Electronics';
+import MensClothing from '../categories/Mens';
+import WomensClothingPage from '../categories/Womens';
+import cart from '../Pages/cart';
+import Address from './Address';
+import Payment from './Payment';
+import Logout from './Logout';
+import PaymentsSuccess from '../component/Payments-success';
+const Main = () => {
+  return (
+    <BrowserRouter
+      future={{
+        v7_startTransition: true,
+        v7_fetcherPersist: true,
+        v7_normalizeFormMethod: true,
+        v7_partialHydration: true,
+        v7_relativeSplatPath: true,
+        v7_skipActionErrorRevalidation: true,
```

Comment:
Please explain me this code.

Reply:
sir this code i will write because if i will get login than it will take me an error of react version update